app/config/routes.js

```diff
@@ -0,0 +1 @@
+// routes?
```

Comment:
I was able to register and login after my change but once I log in nothin is rendering except the boilerplate index.html